src/platform/forms/tests/save-in-progress/SaveFormLink.unit.spec.jsx

```diff
@@ -85,7 +84,7 @@ describe('Schemaform <SaveFormLink>', () => {
     const tree = SkinDeep.shallowRender(
       <SaveFormLink
         user={loggedInUser}
-        form={_.assign(form, { savedStatus: SAVE_STATUSES.failure })}
+        form={{ ...form, savedStatus: SAVE_STATUSES.failure }}
```

Comment:
This is _very_ not worth changing anything here, but for posterity: I once ran some benchmarks on the spread operator vs `Object.assign` (and I think lodash's `assign` too), and the spread operator was surprisingly slow compared to the other two.

It's not going to be even a little bit noticeable here, but I figured I'd mention it. Because JS tidbits are fun. :smile:

Reply:
It's ironic you brought this up, because I was looking into this a few weeks ago and found the same thing to be true. I used [this site](https://www.measurethat.net/Benchmarks/Show/3199/0/lodash-assign-vs-objectassign-vs-spread) when looking into it. The spread operator is definitely slower, which is disappointing 😞. I guess it's a matter of syntactic sugar vs a slight performance boost in cases like these.